ClanGenRun.py

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python3
```

Comment:
Why has main.py been cleared? It seems all it's contents has been moved to this new ClanGenRun.py file. Was there a pressing need to do this?